doc/040_backup.rst

```diff
@@ -276,36 +276,53 @@ suffix the size value with one of ``k``/``K`` for kilobytes, ``m``/``M`` for meg
 Including Files
 ***************
 
-By using the ``--files-from`` option you can read the files you want to back
-up from one or more folders. This is especially useful if a lot of files have
-to be backed up that are not in the same folder or are maybe pre-filtered by
-other software.
+The options ``--files-from``, ``--files-from-verbatim`` and ``--files-from-raw``
+allow you to list the files to back up in one or more files. This is useful
+when a lot of files have to be backed up that are not in the same folder.
```

Comment:
Every time I've read through this documentation I've found this paragraph to be diffuse. I suggest we change it to:

~~~
The options ``--files-from``, ``--files-from-verbatim`` and ``--files-from-raw``
allow you to list files that should be backed up, using one or more files. This is useful
when a lot of files have to be backed up, or when they are not all in the same folder.
~~~

Reply:
I don't think "using" clarifies much. How about:

> The options ``--files-from``, ``--files-from-verbatim`` and ``--files-from-raw`` allow you to list files that should be backed up in a file, rather than on the command line.

Reply:
That's probably a mich clearer description, thank you. I felt that the previous sentence could be read in at least two different ways. Let's change it to your suggestion!

Reply:
Ok, done.